roadmap/implementers-guide/src/node/approval/approval-voting.md

```diff
@@ -192,7 +194,7 @@ On receiving an `ApprovedAncestor(Hash, BlockNumber, response_channel)`:
 
 #### `import_checked_assignment`
   * Load the candidate in question and access the `approval_entry` for the block hash the cert references.
-  * Ensure the validator index is not part of the backing group for the candidate.
+  * Ensure the validator index is not part of the backing group for the candidate.  We count late backing votes via the backing system.
```

Comment:
We don't, at the moment

Reply:
Remove so this represents the current goal, or use future/conditional tense to represent later plans?

Reply:
I suggest removal